tests/README.md

```diff
@@ -15,6 +15,8 @@ You can override the location of the binary using `DRAGONFLY_PATH` environment v
 ### Custom arguments
 
 - use `--gdb` to start all instances inside gdb.
+- use `--df arg=val` to pass custom arguments to all dragonfly instances.
```

Comment:
can you provide a full command instead of a single option?

Reply:
What does full command mean? You can use it multiple times like `--df logtostdout --df proactor_threads=2`, I'll add this info